icons/stackblitz/stackblitz-original.svg

```diff
@@ -0,0 +1 @@
qBLdLoAlbuXmwLF/08Qu0+iAWgHJ6q0Q41u/A/u8zGyWOMf1xcCWJcP2VMRrtWa//0sGS6gLON6buDdEpyadBcpR1sNLMFpmU41phJCp0jSKek5iptv1dPdyssJDjWGP+4uBbGOCV5KzW7wmmmIOtxJ+VKkCOckupLyd4GHXPZKCkSH3oCwlHRFxDYPtxVeMg+9GWjpMh99AvhHxe9AVxxYI8bXvLqO9GXjZIi/QnMJ8bgalD0jyF6hHHr8Eqz68m1sr1Jx8OBxkzERNcoj0whSpn81OsQXcCR33yHx4RvFMsCTbcOz9bpz21wiC7gvDEmxuDKHYFk1ydDlwNbHPVr6TRjJmKizc8xLOl169M7p0lUZUwFpxkzPCa6AzYv6TWa1D22Cl04zRScZoxvhI9FbT4GlwaPrND2bcKbB8gGRxkTjorzQ/Z/hPFWavu9LknLRklxlDF9AWF7l9D6UvKTrTa0wLQMRxnz2YDNMbbqYvLKdpe8VcZUcM65haOiZdi2Z16jBearTtcFnGRMu1/YGGOrKiIvb0WdZTQL55xhyw3bJiQjbWR7T0NrcMhJDoSEb9SGHXIpB48b9m3UHVM2SopDjPnGzwMRGyoDxQXwwkP6EzUSZBnNwgnGTFLoHOXWL9B067B7rf50vdPqAIlxgjF9d/jloNW+EDJdNpIxD5ANTjDGN2LplGQs7TN2TnNe2Sgp0hvjHxfdty3NDLg0eHSldqjJ0p3T8CC9MYN3xbWwpQNMc1X+6gKyGxNj0DnC70YtMoZyaFhCft7szKJ0ishtzK0JcSUorGkDbuhyqNmxRekUkduY4TFxc8KKAcbQxcad0/Ag8fmHo6JzhFuwpJdyqC1xQjjNFCS+BOFJ6ByxIsZWVUT2NOhNyyW+ViYi8VUYDOU8xjZTNtpVnUd1gMTIakw4KlpusIlYbmNsxQVwYJPrybVKl++Q1ZjgPch1sdqtwwsP6U/XK13uQ1Zjum/n9pmXEHi2Tn9+g3xdx3KNlMaEo+LCzVzF2IxH6Wdq9X/YlF9F6RSR0pic7kzh0uDxam3f5rwrSqeIlMZcuJnDGNujK7V9m/K3bJQU+YyJMchRjM2oMjo4nGYK8hnTG+C5iLFRDk3LtP1eJ4fTTEE+Y1qGzH/mNeoAexskboFpGZJdoHBUnL5u/jNvbQk51KTKRikh2TXqHjW/m1CFR4Kd0/Ag2WU6c83MCcmIvr7kzZdwminIdKWuh0W/qZmBwjwLp5mCTMb4Rvjwt8Ks5Q1uHX7UoHRJG2mMmaTQNcqnmDnFaqOnoaoyZoA0xgze5aZMSUaTuvwMp5mCNMZcGhWjkWynJMrBpeVvOM0U5DAmGBGdo/xe1m3AjXDaAa8qG2WOHMb0B4UpyepHV2oHG/M6bZQ9EhgTY3D5jgkxNm+ltnejrDun4UECY+7cE923RSSLzZIoh/pSst+rykYmIMEVHB4T18KZF6uNKuOhZhVOMwfsF3GSwuU7mS/ppRyqisjBRqWLaWC/jqGIaL+V4Us15VBcAC9vdTlg5zQ8YL+Ug3dFX4hnNiUVF8DPGl07lC6mgvpqGntWZ7ak163DgU2uvOppaA2ojQlFRNvNTAYYo8q4e52Kj5gPamM6R/lg+snqmSqj0iUX4DUmxtJe0quqjBaA15hbE2nH2Nw6/GCNM3dOwwNeY9qG03vmjTJ4ZIV2wKvKRrkFrzEtN9L4CEM5bKvSDjUpXXIOUmMGQqLjdqpvSVEGDUvIT7bqShcLQGrMuSE2SVP6m0aV8VCz87fCQgLGqzxJ4fwQ11MYL4yy0aEmVTayDowX+qKfX76b0meYqiLyzw+rcJqlYLzWF29yIZIv0DTCaaoFpsWgu9z+cdEbSP6WVOEhz69XLTBtAJ0xnaN8KFkbcI8bdtdqu9cpXWwAlzGTFPoCIpLwLYmQ6RaYqmxkC7iMMaakBJslUQ7P1OpP16uykW3gMqb7dqIpiXJ4qkb78Rali50gMiYcFV2j/N4CmYEog12rtQNel9LFXhAZ4x9fsNkH5SqchgUsxsQYXBoVw9/GN6ZpmfazraoFJgqwGDMRE52jPG6MzVupWmAiAosxw2PicjBOjK2+lOxt0FXZCA8o7kSMgS/ebmzVxWRPgyob4QLFzZiIie7b98XYjJ6GL252qZ6G2EBxP/qDojtw3/qpQjeoFpg4sf+WxBhc9M9d0qt6GqLFfmMmYqJz5L5i9Z71+gsNqmiEFPuN6Q2I2TtT7FmvH9iidk7Di/2/ymeusSgDnUChezqcporSmLF5jPGPi5Zhbujy2CoVTpMAm43pHOWhSXDrsK1K27dJlY0kwGZjvhjkAMZWWKpsJAd2GjMQEu2jfHM5UWUjibDTmHNDrKSAvLJdhdNkwrZbFY6Kq0HxmtJFNohIsKo2l/QE+EQUVJVROv4fqcUk+nYYcUIAAAAASUVORK5CYII="/></defs><use xlink:href="#a" transform="matrix(.16129 0 0 .11407 49.38 38.89)"/><path style="stroke:none;fill-rule:nonzero;fill:#000;fill-opacity:1" d="M72.898 39.012 49.75 56.91m13.195 0-6.488 11.625m34.176-23.101-.215-.153"/><path style="fill-rule:nonzero;fill:#000;fill-opacity:1;stroke-width:.264583;stroke-linecap:butt;stroke-linejoin:miter;stroke:#000;stroke-opacity:1;stroke-miterlimit:4" d="M119.981 90.842c0 .027-.02.045-.045.045a.043.043 0 0 1-.045-.045c0-.028.02-.055.045-.055.026 0 .045.027.045.055Zm0 0" transform="scale(.6096 .43112)"/></svg>
```

Comment:
This svg has a raster image inside it... so really it's just a .png image inside of a canvas, which we don't allow in devicons.

I suggest you try to find a svg image of the logo.
If there are none, you can view this tutorial for how to convert raster images to vector images: https://github.com/devicons/devicon/issues/1306#issuecomment-1272415252

Once you have the SVG image, make sure to follow our [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) and our requirements for the [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions)